templates/superadmin-footer.html

```diff
@@ -6,10 +6,39 @@
     jQuery.fn.exists = function(){ return this.length > 0; }
 
     $(function() {
-
         $('.single').on('click', function(event){
             event.preventDefault();
-            $('.single_expand').toggle();
+            $(".single").removeClass('active-state');
+            $(this).addClass("active-state");
+            var filter = $(this).attr('id');
+            var page = 1;
+            getSingleSites(page, filter);
         })
+
+        $(document).on('click', '.pagination_link', function(){  
```

Comment:
Maybe add this to the table instead of the document.

Note: [https://api.jquery.com/on/]
Attaching many delegated event handlers near the top of the document tree can degrade performance. Each time the event occurs, jQuery must compare all selectors of all attached events of that type to every element in the path from the event target up to the top of the document. For best performance, attach delegated events at a document location as close as possible to the target elements. Avoid excessive use of document or document.body for delegated events on large documents.

Reply:
Done